src/probnum/diffeq/odefiltsmooth/odefiltsmooth.py

```diff
@@ -176,32 +169,41 @@ def probsolve_ivp(
     >>> from probnum.diffeq import logistic, probsolve_ivp
     >>> from probnum import random_variables as rvs
     >>> import numpy as np
-    >>> initrv = rvs.Constant(np.array([0.15]))
-    >>> ivp = logistic(timespan=[0., 1.5], initrv=initrv, params=(4, 1))
-    >>> solution = probsolve_ivp(ivp, method="ekf0", step=0.1)
+
+    Solve a simple logistic ODE with fixed steps.
+
+    >>> def f(t, x):
+    ...     return 4*x*(1-x)
+    >>>
+    >>> y0 = np.array([0.15])
+    >>> t0, tmax = 0., 1.5
+    >>> solution = probsolve_ivp(f, t0, tmax, y0, step=0.1, adaptive=False)
     >>> print(np.round(solution.y.mean, 2))
     [[0.15]
      [0.21]
      [0.28]
-     [0.36]
-     [0.46]
-     [0.56]
-     [0.65]
+     [0.37]
+     [0.47]
+     [0.57]
+     [0.66]
      [0.74]
      [0.81]
-     [0.86]
-     [0.9 ]
-     [0.93]
-     [0.95]
+     [0.87]
+     [0.91]
+     [0.94]
+     [0.96]
```

Comment:
these values changed, because now the initialisation strategy probsolve ivp  uses RK initialization